.gitignore

```diff
@@ -2,7 +2,7 @@
 
 **/.env
 
-/bin/inx-chronicle/config.toml
+**/config.toml
```

Comment:
Why have this line at all? I would say this is a bit to general.

Reply:
Why? I want to have a local config file without pushing it...

Reply:
But then we should remove the stars. This removes all `config.toml` in all potential directories because of that I think it's too general.